### PR TITLE
Add routes for pages and side-load pages with procedure detail calls

### PR DESCRIPTION
**Pull request by Kapin**

### CODE REVIEW

File: src-backend/api/views.py [hunk 1]
```diff
@@ -1,21 +1,39 @@
...
+    def get_queryset(self):
+        self.resource_name = 'procedure'
+        user = self.request.user
+        return models.Procedure.objects.filter(owner=user)
 
-def versionTestView(request, version):
-    return JsonResponse({'version': __parseVersion(version)})
+    def retrieve(self, request, pk=None):
+        queryset = self.get_queryset()
+        self.resource_name = False
```
Comment (Trinovantes) [inline src-backend/api/views.py]: add a comment why it's false

**Comment by ConnorCimowsky** [comment]
:+1:

File: src-backend/api/views.py [hunk 2]
```diff
@@ -1,21 +1,39 @@
...
 
-class ProcedureViewSet(viewsets.ModelViewSet):
-    serializer_class = ProcedureSerializer
-    model = Procedure
+        data = {
+            'procedure': serial_proc.data,
+            'pages': page_serial.data
+        }
+
+        return Response(data)
```
Comment (Trinovantes) [inline src-backend/api/views.py]: automatically converts to json?

Comment (Kapin) [inline src-backend/api/views.py]: yes

File: src-backend/api/views.py (hunk 1 above)
Comment (Kapin) [inline src-backend/api/views.py]: done

**Comment by Trinovantes** [comment]
+1

**Pull request merged**